### PR TITLE
[TypeSpec-Clean] Update  OpenAI.Authoring

### PR DESCRIPTION
**Pull request by markcowl**
- TypeSpecValidation is failing due to a bug in the tool (validates all folders under cognitiveservices).
- CI Passing: https://dev.azure.com/azure-sdk/public/_build/results?buildId=2772905&view=logs&j=9cd39ef8-47c1-5b58-08db-8c3708712f1b&t=af733ccc-a847-5155-25da-8f7c51f3e136

### CODE REVIEW

File: specification/cognitiveservices/OpenAI.Authoring/tspconfig.yaml [hunk 1]
```diff
@@ -1,7 +1,11 @@
 emit:
   - "@azure-tools/typespec-autorest"
-  - "@azure-tools/cadl-apiview"
-# options:
+#  - "@azure-tools/typespec-apiview"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "cadl-output"
+    emitter-output-dir: "{project-root}"
+    output-file: "{azure-resource-provider-folder}/{version}/openapi.json"
```
Comment (mikeharder) [inline specification/cognitiveservices/OpenAI.Authoring/tspconfig.yaml:8]: Should this output to this folder instead?

https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cognitiveservices/data-plane/AzureOpenAI/authoring

If so, I think this whole directory should be deleted:

https://github.com/Azure/azure-rest-api-specs/tree/main/specification/cognitiveservices/OpenAI.Authoring/cadl-output

**Review by mikeharder — CHANGES_REQUESTED** [review]

File: specification/cognitiveservices/OpenAI.Authoring/tspconfig.yaml (hunk 1 above)
Comment (markcowl) [reply specification/cognitiveservices/OpenAI.Authoring/tspconfig.yaml:8]: It is up to you - the swagger here does not (and never did) match the swagger in the associated data-plane directory.  We could emit it to a different folder in data-plane, but I feel like having the swagger in this directory or removing it altogether are the least bad options.

Comment (mikeharder) [reply specification/cognitiveservices/OpenAI.Authoring/tspconfig.yaml:8]: I'm confused.  If the swagger generated from TypeSpec does not match the swagger in the data-plane directory, why is it being generated at all?  Why does this TypeSpec folder exist in the repo at all?

Comment (markcowl) [reply specification/cognitiveservices/OpenAI.Authoring/tspconfig.yaml:8]: I am guessing that the swagger was there before and this is a proof of concept.  There are multiple versions in the data-plane folder, and this spec does not even represent the latest one.

Comment (mikeharder) [reply specification/cognitiveservices/OpenAI.Authoring/tspconfig.yaml:8]: OK, I'm fine merging this PR as-is with one additional change to remove this file I believe is an extra copy that shouldn't be there:

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/OpenAI.Authoring/cadl-output/openapi.json

**Review by mikeharder — CHANGES_REQUESTED** [review]

File: specification/cognitiveservices/OpenAI.Authoring/tspconfig.yaml (hunk 1 above)
Comment (mikeharder) [reply specification/cognitiveservices/OpenAI.Authoring/tspconfig.yaml:8]: Later we will need to decide what to do with specs that are proof-of-concept or client-only (don't emit swagger).

Comment (weshaggard) [reply specification/cognitiveservices/OpenAI.Authoring/tspconfig.yaml:8]: If this is a proof of concept I would argue we should have it merged into the main branch. If we plan to ship this we should have it outputting specs where they are supposed to be going and should definitely not have the name "cadl-output" in the path.

Comment (mikeharder) [reply specification/cognitiveservices/OpenAI.Authoring/tspconfig.yaml:8]: @lmazuel: Do you know the history of the `OpenAI.Authoring` spec, and who is maintaining it going forward?

Comment (lmazuel) [reply specification/cognitiveservices/OpenAI.Authoring/tspconfig.yaml:8]: Let me ask on our internal chat

**Review by mikeharder — APPROVED** [review]

**Pull request merged**